### PR TITLE
[Ruby] fix port in the Ruby client

### PR DESCRIPTION
**Pull request by wing328**
- fix port in the Ruby client (e.g. https://localhost:3000 will result in the `:3000` omitted in the host in the Ruby client)

### PR checklist
 
- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [x] If contributing template-only or documentation-only changes which will change sample output, [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) beforehand.
- [x] Run the shell script `./bin/generate-samples.sh`to update all Petstore samples related to your fix. This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. These must match the expectations made by your contribution. You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
- [x] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`
- [x] Copy the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

cc @cliffano (2017/07) @zlx (2017/09) @autopp (2019/02)

### CODE REVIEW

**Review by jimschubert — APPROVED** [review]
LGTM

File: modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java [hunk 1]
```diff
@@ -775,6 +775,9 @@ private void generateSupportingFiles(List<File> files, Map<String, Object> bundl
         bundle.put("basePathWithoutHost", basePathWithoutHost);
         bundle.put("scheme", URLPathUtils.getScheme(url, config));
         bundle.put("host", url.getHost());
+        if (url.getPort() != 80 ) {
```
Comment (jirikuncar) [inline modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java:778]: @wing328 shall we consider 443 for HTTPS too?

Comment (wing328) [reply modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java:778]: Sounds good. Do you have time to contribute a PR with your suggestion?

Comment (wing328) [reply modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java:778]: I've filed https://github.com/OpenAPITools/openapi-generator/pull/7531

**Pull request merged**